lib/parser.new.js

```diff
@@ -148,7 +153,7 @@ function Preprocessor (delegate) {
 
   function adjustTag (item) {
     let name = item.name?.toLowerCase () ?? ''
-    if (context.namespace & E.svg)
+    if ((context.namespace ?? 0n) & E.svg)
```

Comment:
Is this needed? If so, that should be a bug, I think? I'd aim for an invariant of context.namespace not being null or undefined. 

But I need to have a closer look at the design of the context object, anyway.